lua/dap.lua

```diff
@@ -50,6 +51,22 @@ M.listeners = {
 }
 
 
+function M.listeners.after.event_initialized.databreakpoints()
+  if session and session.capabilities and session.capabilities.supportsDataBreakpoints then
```

Comment:
could move these also in the respective main functions for the respective events

Reply:
I'd tend to move them into the respective events, given that part of the functionality already requires changes in the built-in event handlers and breakpoints so far are built-in anyways